yb-voyager/src/issue/issue.go

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package issue
+
+type Issue struct {
+	Type            string // (advisory_locks, index_not_supported, etc)
```

Comment:
So, what I realised when adding the issues was that this higher-level type/category like `UNSUPPORTED_FEATURES` / `UNSUPPORTED_QUERY_CONSTRUCTS` is best decided by the consumer, not the issue package. 

For example, consider "Advisory Locks". It can be both under `UNSUPPORTED_QUERY_CONSTRUCTS` or `UNSUPPORTED_STORED_PROCEDURES` (or whatever new type you will assign in your upcoming PR).

Reply:
Hmm okay, makes sense